core/tests/performance_framework/perf_services.py

```diff
@@ -19,17 +19,27 @@
 import os
 import time
 import urlparse
-from browsermobproxy import Server
-from selenium import webdriver
 
 from core.tests.performance_framework import perf_domain
+import feconf
+
+import browsermobproxy
+from selenium import webdriver
 
-CHROME_DRIVER_PATH = os.path.join(
+CHROMEDRIVER_PATH = os.path.join(
     '..', 'node_modules', 'protractor', 'selenium', 'chromedriver_2.21')
 
 BROWSERMOB_PROXY_PATH = os.path.join(
     '..', 'oppia_tools', 'browsermob-proxy-2.1.1', 'bin', 'browsermob-proxy')
 
+# Possible preload options. Only either one of them could be used and they are
+# mutually exclusive.
+DO_LOGIN = 'DO_LOGIN'
```

Comment:
Constants like this should be prefixed with a common 'type', so, e.g.:

    PRELOAD_DO_LOGIN = 'DO_LOGIN'
    PRELOAD_CREATE_EXPLORATION = 'CREATE_EXPLORATION'
    PRELOAD_RELOAD_DEMO_COLLECTIONS = 'RELOAD_DEMO_COLLECTIONS'

etc.

If these names are too long, then maybe

    PRELOAD_DO_LOGIN = 'DO_LOGIN'
    PRELOAD_CREATE_EXP = 'CREATE_EXP'
    PRELOAD_LOAD_DEMO_COLLECTIONS = 'LOAD_DEMO_COLLECTIONS'